src/abstract_time_series.jl

```diff
@@ -32,7 +36,7 @@ abstract type AbstractTimeSeriesParameters <: InfrastructureSystemsType end
 struct StaticTimeSeriesParameters <: AbstractTimeSeriesParameters end
 
 @kwdef struct ForecastParameters <: AbstractTimeSeriesParameters
```

Comment:
A variation of this struct has existed for years and always stored resolution and interval as Dates.Period. All of our forecast data and metadata structs have always stored them as Dates.Period. The abstract type is more flexible. What improvement would result in coercing to Millisecond? The only time the type matters is when we perform division and it can be converted as necessary.

Reply:
that's ok, I was just wondering.